addons/base_import_module/models/ir_module.py

```diff
@@ -363,6 +363,8 @@ def _get_modules_from_apps(self, fields, module_type, module_name, domain=None,
             resp = self._call_apps(json.dumps(payload))
             resp.raise_for_status()
             modules_list = resp.json().get('result', [])
+            if not modules_list:
+                raise UserError(_('The list of industry applications for %s is empty.', module_name))
```

Comment:
As discussed, I wouldn't add this error. The change in web_read solves the issue + this may break custom flow relying on this method.

Reply:
@aj-fuentes I think that we can indeed remove the UserError. Otherwise the PR looks good to me.

Reply:
Removed it.